postgres/datadog_checks/postgres/metrics_cache.py

```diff
@@ -197,13 +215,20 @@ def get_activity_metrics(self, version):
             else:
                 metrics_query = ACTIVITY_METRICS_LT_8_3
 
+            default_descriptors = [('datname', 'db'), ('application_name', 'application_name')]
```

Comment:
So for dbm activity metrics that we generate from pg_stat_activity data, such as `postgresql.connections`, the tag name we use for applications is `app` e.g. [here](https://app.datadoghq.com/metric/explorer?start=1673025988735&end=1673029588735&paused=false#N4Ig7glgJg5gpgFxALlAGwIYE8D2BXJVEADxQEYAaELcqyKBAC1pEbghkcLIF8qo4AMwgA7CAgg4RKUAiwAHOChASAtnADOcAE4RNIKtrgBHPJoQaUAbVBGN8qVoD6gnNtUZCKiOq279VKY6epbINiAiGOrKQdpYZAYgUJ4YThr42gDGSsgg6gi6mZaBZnHKGABuMMgOGggwdsZoAHSZUiJwmRKOwHAiFchQAEaqALQaWCJMiBBFPAAEQ1jzwBjy8jwgPAC6VK7ueJih4fuqhxgxpfFbuyB1WGg5oPIYjwgIOUk4ME6ZRxqzRJoURwJxyRTKdLAqBAkFOehMZQiNweNBbfgQeyYLBghSfYEdG48Ph3eTAhAAYSkwhgKBEhzQPCAA) 

I think it makes sense to try to keep these consistent across DBM and regular integration data